### DISCUSSION TITLE
THT Reference Manual is complete

### DISCUSSION THREAD

**Discussion by aoirthoir** [joelesko/tht#23] [General]
I have collected and organized the entire website and created a 1040 page Reference manual. It does not have an index, or clickable TOC. It maintains mostly the look of the website. I do not have copyright information in it yet, but that can all be added. 

So what should I do next? Would the team behind THT like me to upload it somewhere?

**Comment by joelesko** [comment]
The team is just me. :)  You can feel free to post it wherever you want publicly. 

Just make a note that it's a community generated document and that it might be out of sync with later versions of the site.

**Comment by aoirthoir** [comment]
Well I am not up on my programming skills enough to create a full on programming language  but i'll add myselph to the documentation team for now.  I will add it here to the discussion so you can review.


[THT-Reference-Manual-Output.pdf](https://github.com/joelesko/tht/files/12797009/THT-Reference-Manual-Output.pdf)
